src/models/decorators.ts

```diff
@@ -66,10 +67,50 @@ export function isColumnNullable(targetClass: Function, propertyName: string): b
     return false;
 }
 
-export function getPrimaryKey(targetClass: Function): string | symbol | undefined {
+export function getPrimaryKeys(targetClass: Function): string[] {
     const metadata = metadataRegistry.get(targetClass);
     if (metadata) {
-        return metadata.primaryKey;
+        // Get all the keys of the metadata object that are primary keys
+        const keys = Object.keys(metadata.columns).filter((key) => metadata.columns[key].primary);
+
+        // Return the actual column name as it is stored in the database
+        return keys.map((key) => metadata.columns[key].name);
+    }
+    return [];
+}
+
+/**
+ * Based on the column name value, however it is stored in the database, get the actual
+ * key name of the property in the class.
+ * 
+ * @param targetClass 
+ * @param columnName 
+ * @returns String – the property name
+ */
+export function getColumnValueFromName(targetClass: Function, columnName: string): string {
+    const metadata = metadataRegistry.get(targetClass);
+    if (metadata) {
+        for (const key in metadata.columns) {
+            if (metadata.columns[key].name === columnName) {
+                return key;
+            }
+        }
+    }
+    return '';
+}
+
+/**
+ * Based on the actual property name, usually camel cased, get the column name value
+ * that is stored in the database.
+ * 
+ * @param targetClass 
+ * @param propertyName 
+ * @returns String – the column name
+ */
+export function getColumnValueFromProperty(targetClass: Function, propertyName: string): string {
+    const metadata = metadataRegistry.get(targetClass);
+    if (metadata) {
+        return metadata.columns[propertyName].name;
     }
-    return undefined;
+    return '';
```

Comment:
Do you think there's a benefit to returning a `''` instead of `null`? I feel like returning an empty string has bitten me before when I'm not sure why something is wrong. By having `null` potentially be the return type, the IDE kind of guides me in a way to say "Hey, this can return `null` which might impact." The empty string doesn't really provide any of that feedback for a developer.